traefik/values.yaml

```diff
@@ -91,8 +91,8 @@ deployment:
   # -- Pod lifecycle actions
   lifecycle: {}
   # preStop:
-  #   exec:
-  #     command: ["/bin/sh", "-c", "sleep 40"]
+  #   sleep:
+  #     seconds: 20
   # postStart:
```

Comment:
Wdyt of removing this and transform it into an example in EXAMPLES.md ?

Reply:
To be fair, though there may be a viable purpose for this in practice, I think generally using the `requestAcceptGraceTimeout` (configured for all ports) is a better way to ensure non-disrupted traffic handling with different kind of load balancers. So I think if anything we should probably document that and just leave this as a reference for specific needs (which I can't currently come up with).